_data/toc.yaml

```diff
@@ -365,6 +365,8 @@ guides:
       title: Volumes
     - path: /storage/bind-mounts/
       title: Bind mounts
+    - path: /docker-for-windows/mounts-on-windows/
```

Comment:
For me it feels like the page path should be `/storage/mounts-on-windows/` as it's listed in the storage section and not in the docker-for-windows section.
@usha-mandya WDYT?
I understand that this page is specific to Docker Desktop for Windows, so it may be listed there as well? 🤔

Reply:
Agree. We can add a link to this page from the Docker Desktop for Windows page.

Reply:
I'm a bit uncertain about whether I am supposed to change something at this stage. To me it looks like you're still in an internal discussion?

I'll gladly change things to your discretion (or leave it up to you, of course). Just address me imperatively when you feel it's time I can do something to amend the PR.